pyproject.toml

```
@@ -0,0 +1,121 @@
+[tool.poetry]
+name = "udata"
+version = "4.1.3-alpha.2"
+description = "Open data portal"
+readme="README.md"
+authors = ["Opendata Team <opendatateam@data.gouv.fr>"]
```

Should we use Etalab? and udata@data.gouv.fr?
```suggestion
authors = ["Opendata Team <opendatateam@data.gouv.fr>"]
```